substra/cli/interface.py

```diff
@@ -491,6 +534,67 @@ def add_traintuple(ctx, objective_key, algo_key, dataset_key, data_samples, in_m
     printer.print(res, is_list=False)
 
 
+@add.command('composite_traintuple')
+@click.option('--objective-key', required=True)
+@click.option('--algo-key', required=True)
+@click.option('--dataset-key', required=True)
+@click.option('--data-samples-path', 'data_samples', required=True,
+              type=click.Path(exists=True, resolve_path=True, dir_okay=False),
+              callback=load_json_from_path)
+@click.option('--head-model-key',
+              help='Must be used with --trunk-model-key option.')
+@click.option('--trunk-model-key',
+              help='Must be used with --head-model-key option.')
```

Comment:
The doc says these must be used together but I don't see a check enforcing this rule. Could you add one?